tests/Unit/Time/TimeSteppers/Test_RungeKutta3.cpp

```diff
@@ -16,6 +16,10 @@ SPECTRE_TEST_CASE("Unit.Time.TimeSteppers.RungeKutta3", "[Unit][Time]") {
   TimeStepperTestUtils::integrate_test(stepper, 0, -1., 1e-9);
   TimeStepperTestUtils::integrate_test_explicit_time_dependence(stepper, 0,
                                                                 -1.0, 1.0e-9);
+  // The RK3(2) scheme is lower order, so has more disparity in the error
+  // estimate
```

Comment:
Lower order than what?  This looks like it has the tightest error bound of any of the tests.